### DISCUSSION TITLE
Adding Bluetooth

### DISCUSSION THREAD

**Discussion by CptKnight127** [gpstar81/GPStar-proton-pack#267] [General]
Hello All,
Just wanted to ask if anybody in the community has tried adding Bluetooth capability to the pack. I'm interested in connecting my phone up to it and play more music that I have saved to my YT Music or Amazon Music playlists. Any help is appreciated.

**Comment by Toy203** [comment]
I solved this by adding a secondary aux in on the pack. Got a tiny passive 2way mixer as Dustin suggested and an aux in port on the back of the motherboard. With this i'm able to plug in any receiver (got a sd/bt reader from aliexpress) and it feed directly to the amp so i can play whatever i want through the speakers, as long as the pack has main power (even when it is turned off of course).

**Comment by CptKnight127** [comment]
Thank you for the info. I'll look into this some more.

**Comment by Toy203** [comment]
Let me know if you need more info

**Comment by DustinGrau** [comment]
As noted, this is as simple as adding a mixer device then allowing either a line-in from your phone or using an off-the-shelf bluetooth device. Doesn't need to be a kit solution, just roll your own mix of inputs to the pack's amplifier.

Bluetooth Receiver: https://a.co/d/cEX0LzW

2-to-1 Passive Mixer: https://a.co/d/16Srz1B
